.github/workflows/build.yml

```diff
@@ -8,15 +8,15 @@ on:
 jobs:
 
   linux:
-    runs-on: ubuntu-22.04
```

Comment:
Sorry, I missed this: this was deliberately set to 22.04. Can you undo this change?